app/src/test/java/com/fstyle/structure_android/data/source/UserRepositoryTest.java

```diff
@@ -67,43 +64,26 @@ public void searchUsers200ResponseCodeInvokesCorrectApiCalls() {
         List<User> users = onNextEvents.get(0);
         Assert.assertEquals(USER_LOGIN_1, users.get(0).getLogin());
         Assert.assertEquals(USER_LOGIN_2, users.get(1).getLogin());
-        Mockito.verify(mNameApi).searchGithubUsers(2, USER_LOGIN_1);
-    }
-
-    private UsersList githubUserList() {
-        User user = new User(USER_LOGIN_1);
-
-        User user2 = new User(USER_LOGIN_2);
-
-        List<User> githubUsers = new ArrayList<>();
-        githubUsers.add(user);
-        githubUsers.add(user2);
-        UsersList usersList = new UsersList(githubUsers);
-        return usersList;
+        Mockito.verify(mRemoteDataSource).searchUsers(USER_LOGIN_1, 2);
     }
 
     @Test
-    public void searchUsersOtherHttpErrorSearchTerminatedWithError() {
+    public void searchUsers_OtherHttpError_SearchTerminatedWithError() {
```

Comment:
ERROR: [MethodNamingConventions] Method names should not contain underscores
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#MethodNamingConventions